modules/cloudsql-instance/main.tf

```diff
@@ -69,10 +69,11 @@ resource "google_sql_database_instance" "primary" {
     time_zone                   = var.time_zone
 
     ip_configuration {
-      ipv4_enabled       = var.network_config.connectivity.public_ipv4
-      private_network    = try(var.network_config.connectivity.psa_config.private_network, null)
-      allocated_ip_range = try(var.network_config.connectivity.psa_config.allocated_ip_ranges.primary, null)
-      ssl_mode           = var.ssl.ssl_mode
+      ipv4_enabled                                  = var.network_config.connectivity.public_ipv4
+      private_network                               = try(var.network_config.connectivity.psa_config.private_network, null)
+      allocated_ip_range                            = try(var.network_config.connectivity.psa_config.allocated_ip_ranges.primary, null)
+      ssl_mode                                      = var.ssl.ssl_mode
+      enable_private_path_for_google_cloud_services = var.network_config.connectivity.enable_private_path_for_google_cloud_services
```

Comment:
Can we please also have this new argument for the replicas setting (L #185)? Would be great to query the read replica from BigQuery

Reply:
added to the replicas as well!